department-of-veterans-affairs/va.gov-cms
# VACMS-2824: Add patched flood_control module & configure user block expiration

## Description

See #2824 and #2823

## Testing done

Manual testing. 

## Screenshots

<img width="1003" alt="Screen Shot 2021-01-19 at 4 44 16 PM" src="https://user-images.githubusercontent.com/101649/105107532-94db3700-5a75-11eb-97ec-7e9599e6a087.png">

## QA steps

Attempt to login with a valid username and incorrect password more than 5 times in a row. You should see a message that the account has been temporarily blocked. Then, review the [new documentation](https://github.com/acrollet/va.gov-cms/blob/VACMS-2824-2825-flood_control/READMES/user_management.md) and log in as a user with the `Administrator` role:

- [ ] Confirm that the user login flood entries are set to expire 30 minutes after the login attempt was made
- [ ] The blocked user account can be manually unblocked

### Definition of Done


### Select Team for PR review


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
